Havit.Blazor.Components.Web.Bootstrap/Forms/SearchBox/HxSearchBox.razor.cs

```diff
@@ -274,6 +281,8 @@ protected async Task UpdateSuggestionsAsync()
 		}
 
 		dataProviderInProgress = false;
+
+		focusedItemIndex = default;
```

Comment:
Do not use `default` where the value is known in compile time and is not subject to later change in type implementation.

Reply:
Forgot to adjust this - I started with an implementation where I was storing focusedItem (type TItem) and not the index, therefore I had to use default, thanks :)